src/__Plugin.ino

```diff
@@ -1200,14 +1200,46 @@ byte PluginCall(byte Function, struct EventStruct *event, String& str)
     case PLUGIN_WRITE:
     case PLUGIN_REQUEST:
     {
-      for (taskIndex_t task = 0; task < TASKS_MAX; task++)
+      taskIndex_t firstTask = 0;
+      taskIndex_t lastTask = TASKS_MAX;
+      String command = String(str);                         // Local copy to avoid warning in ExecuteCommand
+      int dotPos = command.indexOf(".");                    // Find first period
+      if (Function == PLUGIN_WRITE                          // Only applicable on PLUGIN_WRITE function
+        && dotPos > -1) {                                   // First precondition is just a quick check for a period (fail-fast strategy)
+        String thisTaskName = command.substring(1, dotPos); // Extract taskname
```

Comment:
This is not correct.
The parameter separater we always used is still valid here.
`[name].command<comma>param1...`
So please use the `parseString` to find the proper substring first.  (is in StringConverter.cpp)
On this substring (the first argument, sadly not counting from 0) is the command with the optional task index information.

On this substring you can test for a dot and try to find the task index from that information.
If you have the task index, call the function I describe in a comment below.